lom/src/lib/lomcommon/config.go

```diff
@@ -314,6 +361,15 @@ func (p *ConfigMgr_t) loadConfigFiles(cfgFiles *ConfigFiles_t) error {
     if err := p.readBindingsConf(cfgFiles.BindingsFl); err != nil {
         return LogError("Bind: %s: %v", cfgFiles.BindingsFl, err)
     } 
+    test_mode_fl := filepath.Join(filepath.Dir(cfgFiles.BindingsFl), LOM_TESTMODE_NAME)
```

Comment:
Why is it checking LOM_TESTMODE_NAME ?

Reply:
It is looking for file by name "LomTesMode" as initialized in this constant.